Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM ubuntu:14.04
+MAINTAINER Cezary Baginski <cezary@chronomantic.net>
```

Comment:
Please replace this with

~~~
MAINTAINER Aruba Maintainers <cukes-devs@googlegroups.com>
~~~